docker-compose.yml

```diff
@@ -117,10 +117,10 @@ services:
       JAVA_OPTS: -server -Djava.security.egd=file:/dev/./urandom -Xms1024m -Xss512k -Xmx1024m -XX:TieredStopAtLevel=1
       FLYWAY_USERNAME: ${EVAKA_SERVICE_FLYWAY_USERNAME}
       FLYWAY_PASSWORD: ${EVAKA_SERVICE_FLYWAY_PASSWORD}
-      FLYWAY_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/${EVAKA_SERVICE_DATABASE_NAME}
+      FLYWAY_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/evaka_tampere_local
       FLYWAY_PLACEHOLDERS_MIGRATION_USER: ${EVAKA_SERVICE_FLYWAY_USERNAME}
       FLYWAY_PLACEHOLDERS_APPLICATION_USER: ${EVAKA_SERVICE_APPLICATION_USERNAME}
-      SPRING_DATASOURCE_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/${EVAKA_SERVICE_DATABASE_NAME}
+      SPRING_DATASOURCE_URL: jdbc:postgresql://${EVAKA_DATABASE_HOST}:${EVAKA_DATABASE_PORT}/evaka_tampere_local
```

Comment:
`evaka_tampere_local` -> `evaka_${EVAKA_CUSTOMIZATIONS}_local` should work for all.

Reply:
Changed and seems to be working